### PR TITLE
Template merge 1.11 

### PR DESCRIPTION
**Pull request by Zethson**
Signed-off-by: Zethson <lukas.heumos@posteo.net>

Did not want to work on the TEMPLATE branch -> here's the merge for 1.11

### CODE REVIEW

File: conf/base.config [hunk 1]
```diff
@@ -22,32 +22,24 @@ process {
...
   // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
-  withLabel:process_very_low {
-    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
-    memory = { check_max( 6.GB * task.attempt, 'memory' ) }
-    time = { check_max( 3.h * task.attempt, 'time' ) }
-  }
   withLabel:process_low {
-    cpus = { check_max( 4 * task.attempt, 'cpus' ) }
-    memory = { check_max( 10.GB * task.attempt, 'memory' ) }
+    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
```
Comment (Zethson) [inline conf/base.config:26]: Are those nf-core wide defaults or defined by us?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -11,12 +10,13 @@ on:
...
 jobs:
   test:
+    name: Run workflow tests
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/proteomicslfq') }}
+    runs-on: ubuntu-latest
     env:
       NXF_VER: ${{ matrix.nxf_ver }}
       NXF_ANSI_LOG: false
-      TOWER_ACCESS_TOKEN: ${{ secrets.NONAWS_TOWER_ACCESS_TOKEN }}
```
Comment (Zethson) [inline .github/workflows/ci.yml:17]: Is this supposed to be deleted?

Comment (Zethson) [reply .github/workflows/ci.yml:17]: I will readded it for now.

**Comment by Zethson** [comment]
```
x Test failures:

    Test #5 - CI is not building the correct docker image. Should be: docker build --no-cache . -t nfcore/proteomicslfq:1.0.0
    Test #5 - CI is not tagging docker image correctly. Should be: docker tag nfcore/proteomicslfq:dev nfcore/proteomicslfq:1.0.0

exclamation Test warnings:

    Test #4 - Config manifest.version should end in dev: '1.0.0'

```

@jpfeuffer should we ignore them, since we want to continue with `dev`?

File: .github/workflows/awstest.yml [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
 # This workflow is triggered on push to the master branch.
-# It runs the -profile 'test' on AWS batch
+# It can be additionally triggered manually with GitHub actions workflow dispatch.
+# It runs the -profile 'test' on AWS batch.
 
 on:
-  push:
-    branches:
-      - master
+  workflow_dispatch:
```
Comment (jpfeuffer) [inline .github/workflows/awstest.yml:7]: no master?

File: conf/base.config (hunk 1 above)
Comment (jpfeuffer) [reply conf/base.config:26]: I think we overwrote them

Comment (jpfeuffer) [reply conf/base.config:26]: we defo need single_threaded and very_low

**Review by jpfeuffer — COMMENTED** [review]
thanks!

**Comment by jpfeuffer** [comment]
> ```
> x Test failures:
> 
>     Test #5 - CI is not building the correct docker image. Should be: docker build --no-cache . -t nfcore/proteomicslfq:1.0.0
>     Test #5 - CI is not tagging docker image correctly. Should be: docker tag nfcore/proteomicslfq:dev nfcore/proteomicslfq:1.0.0
> 
> exclamation Test warnings:
> 
>     Test #4 - Config manifest.version should end in dev: '1.0.0'
> ```
> 
> @jpfeuffer should we ignore them, since we want to continue with `dev`?

What are the options?
call it dev? call it 1.1.0pre or something? why would we stick with 1.0.0?

**Comment by jpfeuffer** [comment]
Regarding new release cycle: https://github.com/nf-core/proteomicslfq/pull/99
Which I forgot to merge.

File: .github/workflows/awstest.yml (hunk 3 above)
Comment (Zethson) [reply .github/workflows/awstest.yml:7]: https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/workflows/awsfulltest.yml#L6

Not in the template.
The `workflow_dispatch` allows you to trigger it manually. 
Hence, you can trigger it once before releasing and after the release has been published it gets triggered anyways.

**Comment by Zethson** [comment]
@jpfeuffer I readded the single threaded and very low configs.

If you want I can change the other profiles to the former values. Whatever you prefer.

I think I got confused between nf-core's standards and what I personally follow at [mlf-core](https://mlf-core.com/index). I personally bump to *-SNAPSHOT versions, whose nf-core equivalent is *dev. Hence, the new version is now 1.1.0dev. After we have merged this you can merge the documentation PR that you mentioned.

Cheers

File: conf/base.config (hunk 1 above)
Comment (Zethson) [reply conf/base.config:31]: Readded both.

Can revert the changes for the other ones if you want me to.

**Comment by jpfeuffer** [comment]
I see. Yeah 1.1.0dev sounds good. I guess the Docker containers are still tagged :dev?
I am in favor of using our old default memory/CPU settings.
On the one hand I don't want to always adapt them at template updates, on the other I like ours better.
The nfcore defaults e.g. feel a bit disproportional with CPU vs Mem

**Comment by Zethson** [comment]
```
      - name: Push Docker image to DockerHub (dev)
        if: ${{ github.event_name == 'push' }}
        run: |
          echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
          docker tag nfcore/proteomicslfq:latest nfcore/proteomicslfq:dev
          docker push nfcore/proteomicslfq:dev
```

Yes, they are still tagged dev. I used the `nf-core bump-version` command, so everything is as it should be.

**Comment by Zethson** [comment]
@jpfeuffer reverted configs to the previous ones.

Ready to merge from my side.

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**